cuda2gcn/src/lanemask.ll

```diff
@@ -0,0 +1,45 @@
+target datalayout = "e-p:64:64-p1:64:64-p2:64:64-p3:32:32-p4:64:64-p5:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-v2048:2048-n32:64"
+target triple = "amdgcn--amdhsa"
+
+; Function Attrs: alwaysinline nounwind
+declare i32 @__ockl_activelane_u32() #0
+
+; Function Attrs: alwaysinline nounwind
+define i64 @__gcn_ballot64(i32 %a) #0 {
```

Comment:
Implement this in C using __llvm_amdgcn_icmp_i32 from irif.h.  The last argument indicates which comparison.  ICMP_EQ=32, ICMP_NE=33, ICMP_UGT=34...  See InstrTypes.h in include/llvm/IR